### ISSUE TITLE
Gradle and Readme

### ISSUE THREAD

**Issue by ftomassetti** [IETS3/iets3.opensource#4]
The Readme refers to the ant script but a gradle script is present in the root of the project.
What is its intended usage? Should we explain what it is in the Readme?

**Comment by sergej-koscejev** [comment]
Gradle scripts are work in progress for now.

**Comment by levilucio** [comment]
I used Gradle to build on a Mac. The build is successful but when I run the mps.sh script I get this error:

```
/usr/bin/readlink: illegal option -- f
usage: readlink [-n] [file ...]
./mps.sh: line 71: message: command not found
```

The script launches MPS successfully if ran on a linux.

Should I not use gradle to build on a mac, but just the ant scripts?

**Comment by sergej-koscejev** [comment]
I don't see `mps.sh` script mentioned anywhere in the build instructions so I'm confused. What are you trying to do and where is this `mps.sh` coming from?

**Comment by levilucio** [comment]
When I build iets3.opensource using gradle MPS gets copied inside the build. I am trying to run it using:

iets3.opensource/build/mps/bin/mps.sh

but I get the error above.
Is this not the correct way of running MPS after gradle finishes the build? It runs fine on a linux as I mentioned...

**Comment by sergej-koscejev** [comment]
No, it's not the correct way, just use your own installed version of MPS and open the project.

**Comment by sergej-koscejev** [comment]
Since README now mentions Gradle I'm closing this issue.

**Comment by davidmpaz** [comment]
Hi @sergej-koscejev,
> No, it's not the correct way, just use your own installed version of MPS and open the project.

and then? ... I can build the **iets3.opensource** project using gradle, get a couple of exceptions but at end build finish in success. But then I get lost since don know what artifacts to use.

Tried:
* Use ZIP file in `build/distributions/` to import as plugin into MPS, did not worked.
* Trying to import languages, in a newly created project. No success either. I suppoused here that build could install somehting into my installed MPS.
* Tried to execute (run) models `org.iets3.opensource.allScripts.build.build.org.iets3.opensource.allScripts.build` and `org.iets3.opensource.build.build.org.iets3.opensource` but get errors too

all of this is under:
JetBrains MPS 2017.3.5
Build #MPS-173.1767, built on February 15, 2018
JRE: 1.8.0_161-b12 amd64
JVM: Java HotSpot(TM) 64-Bit Server VM by Oracle Corporation
Linux 4.13.0-37-generic

thanks in advance,
David
